plugins/labhub.py

```diff
@@ -108,6 +89,21 @@ def team_mapping(self):
             'maintainers': self.TEAMS[self.GH_ORG_NAME + ' maintainers'],
         }
 
+    @cmdfilter
+    def members_only(self, msg, cmd, args, dry_run):
+        user = msg.frm.nick
+        commands = ['assign_cmd', 'create_issue_cmd', 'invite_cmd', 'mark_cmd',
+                    'pr_stats', 'unassign_cmd', 'pitchfork', 'the_rules', 'wa',
+                    'answer', 'lmgtfy', 'ghetto', 'explain', 'nevermind']
+
+        if cmd in commands:
+            for team in self.team_mapping().values():
+                if team.is_member(user):
+                    return msg, cmd, args
+            return None, None, None
```

Comment:
cant we also emit a message `'You need to be a member of this organization to use this command.'` so that the tests dont change?

perhaps change it to `return None, 'echo', 'You need to be a member of this organization to use this command.'` and add an `echo` command?

Reply:
Okay, I'll take care of this.